op-chain-ops/genesis/config.go

```diff
@@ -26,6 +26,11 @@ import (
 	"github.com/ethereum-optimism/optimism/op-service/eth"
 )
 
+// initialzedValue represents the `Initializable` contract value. It should be kept in
+// sync with the constant in `Constants.sol`.
+// https://github.com/ethereum-optimism/optimism/blob/b6104508cb11ac85252484e212c199acf512a874/packages/contracts-bedrock/src/libraries/Constants.sol#L49
```

Comment:
Should we use a regular link to `Constants.sol` instead of a permalink? Linking to this commit may make it look like `3` is always the most up-to-date value

Reply:
That is a great catch